### PR TITLE
Tailwindify van holiday elements

### PR DESCRIPTION
**Pull request by RemcoSwenker**

### CODE REVIEW

File: config/components/config/holiday.py [hunk 1]
```diff
@@ -0,0 +1,12 @@
+from django_components import Component, register
+
+
+@register("config.holiday_table")
+class HolidayTable(Component):
+
+    # This component takes one parameter, a date string to show in the template
```
Comment (nanderv) [inline config/components/config/holiday.py:7]: Haha, this comment is still getting copy-pasted :)

File: config/components/config/holiday.py [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+    def get_context_data(self, holidays=""):
```
Comment (nanderv) [inline config/components/config/holiday.py:8]: I'd use 'None' as default value then, perhaps with a type-hint as well.

File: config/components/config/holiday_table.html [hunk 3]
```diff
@@ -0,0 +1,31 @@
+{% component 'tile/tile' root_class_tags="grow max-w-screen lg:basis-2/5  lg:flex-none" %}
+    {% fill 'title' %}Holidays{% endfill %}
+    {% fill 'content' %}
+        <table class="table table-striped" style="width: auto;">
```
Comment (nanderv) [inline config/components/config/holiday_table.html:4]: I thought it's table-zebra in tailwind/daisy? Also, I think width: auto is no longer required.

**Review by nanderv — COMMENTED** [review]
Looks good, some comments.
Also, you can import the datepicker directly in `base_header.html`, and then we have it everywhere.
At a later point, we probably want to make our datepicker a component.

File: config/components/config/holiday.py (hunk 1 above)
Comment (RemcoSwenker) [reply config/components/config/holiday.py]: right well got rid of it ;)

File: config/components/config/holiday.py (hunk 2 above)
Comment (RemcoSwenker) [reply config/components/config/holiday.py]: fixed the type hint

**Review by nanderv — APPROVED** [review]

**Pull request merged**